source/fundamentals/connection.txt

```diff
@@ -19,6 +19,15 @@ Connection Guide
 This guide shows you how to connect to a MongoDB instance or replica set
 deployment using the Go Driver.
 
+-------------
+Compatibility
```

Comment:
Q: Why is this included on the Go connection page, but not C#/Java?

Reply:
So currently, we're working on an Atlas 250 project where we're trying to make the top 250 ranked pages in docs to be more Atlas centric. The connection page for C# and Java didn't make the cut 🥲

Reply:
yeah, i noticed the two jira tickets right after i sent back to you